dev/start

```diff
@@ -179,6 +230,11 @@ fi
 if [[ $ENABLE_AUTHN_JWT = true ]]; then
   enabled_authenticators="$enabled_authenticators,authn-jwt/raw"
   services="$services jwks"
+  docker-compose up -d --no-deps $services
+
+  # OIDC is a special case on JWT, JWT automation tests contain scenarios with OIDC providers
+  configure_oidc_providers
+  enable_oidc_authenticators
```

Comment:
this should go inside `configure_oidc_providers` right after `configure_oidc_authenticators`

Reply:
Done